internal/restic/tag_list.go

```diff
@@ -40,6 +40,23 @@ func (l TagLists) String() string {
 	return fmt.Sprintf("%v", []TagList(l))
 }
 
+// Flatten returns the list of all tags provided in TagLists
+func (l TagLists) Flatten() *TagList {
```

Comment:
Please return the `TagList` as value and not a pointer. This also allows removing the `len(l)==0` special case.